### ISSUE TITLE
PPO2 on Swimmer-v2. Avg reward plot matches the one in the paper but video is not meaningful.

### ISSUE THREAD

**Issue by hiteshisharma** [openai/baselines#1013]
I ran PPO2 on Mujoco Swimmer-v2 with the hyper-parameters as mentioned in the paper. I was able to generate the plot for avg rewards similar to the one in the paper but when I ran the trained model to visualize, the agent did not seem to working. 
![plot_ppo2_Swimmer-v2](https://user-images.githubusercontent.com/50846888/66072273-9d6acf80-e509-11e9-8707-e644e109aeec.jpg)
![swimmer](https://user-images.githubusercontent.com/50846888/66072823-a8722f80-e50a-11e9-9f04-c8718d82fc38.gif)

Has anyone faced the same issue?

**Comment by christopherhesse** [comment]
Assuming you're not seeing the correct average reward when visualizing the agent, are you failing to load the parameters or using different observation normalization?

**Comment by hiteshisharma** [comment]
It looks like that in order to have a working agent in the visualization, the swimmer (v2) environment should reach an avg reward of ~300 but it is stuck around 100-110. So, while the avg reward is same while visualization, it does not seem to be moving. 

Please let me know if I am not understanding it fully.

**Comment by christopherhesse** [comment]
Oh, so you're saying the learning curve is consistent with the video, but that the learning curve isn't very good because it should get to a reward of 300, but only gets to like 100, right?

**Comment by christopherhesse** [comment]
It looks like baselines expects to get about 100 for swimmer: http://htmlpreview.github.io/?https://github.com/openai/baselines/blob/master/benchmarks_mujoco1M.htm 

Where did you get the 300 value from?

**Comment by hiteshisharma** [comment]
This paper :  [https://arxiv.org/pdf/1906.08649.pdf] 

What I meant was that we need more than 100-120 to see the swimmer working in the video.

**Comment by christopherhesse** [comment]
Looking at that paper, it seems to say that PPO gets a score of 155 after 200k timesteps, right?
